nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/Repeat.java

```diff
@@ -59,6 +59,14 @@ public Repeat(INDArray[] inputs, INDArray[] outputs, int axis) {
         super(null, inputs, outputs);
         this.jaxis = axis;
     }
+    public Repeat(INDArray[] inputs, int axis) {
+        this(inputs, null, axis);
+    }
```

Comment:
Repeat only allows single input array - delete INDArray[] constructor

Reply:
Deleted. Makes the Repeat(INDArray input, int axis) a todo: implement.